changelog.d/14146.removal

```diff
@@ -0,0 +1 @@
+Remove the unstable identifier for [MSC3715](https://github.com/matrix-org/matrix-doc/pull/3715).
```

Comment:
Don't mind us reusing the changelog, provided #14106 didn't land in 1.69. (erm, did it?)

Reply:
It didn't land in 1.69.